gentoopm/basepm/upstream.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+# 	vim:fileencoding=utf-8
+# (c) 2024 Anna <cyber@sysrq.in>
+# Released under the terms of the 2-clause BSD license.
+
+from dataclasses import dataclass
+from enum import Enum, auto
+
+
+class PMUpstreamMaintainerStatus(Enum):
+    """ Maintainer status enumeration. """
```

Comment:
Please try to follow PEP for docstrings, or at least keep the style consistent.

```suggestion
    """Maintainer status enumeration"""
```